### PR TITLE
SASS ALL THE THINGS

### PR DESCRIPTION
**Pull request by alexmckenley**
@deini @malucky Please review

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -63,8 +63,8 @@ gulp.src(mainBowerFiles(), { base: 'bower_components' })
  * Concat css and move to build dir.
  */
 gulp.task('styles', function () {
-  gulp.src(paths.styles)
-    .pipe(concatCss('bundle.css'))
+  gulp.src('public/sass/*.scss')
```
Comment (malucky) [inline gulpfile.js]: throw this into a variable for consistency :tropical_drink:

File: public/sass/shame/_entire-site.scss [hunk 2]
```diff
@@ -99,7 +96,7 @@ a {
 }
 
 .sidebar {
-  width: 20%%;
```
Comment (malucky) [inline public/sass/shame/_entire-site.scss:102]: niceee

Comment (malucky) [inline public/sass/shame/_entire-site.scss:102]: 100%%

File: gulpfile.js (hunk 1 above)
Comment (deini) [inline gulpfile.js]: Should it be `public/sass/**/*.scss`?

**Comment by malucky** [comment]
i feel like there could be better folder name than javascript because the templates are in there too and this could be unintuitive for anyone not familiar with the project, but i can't really think of a better alternative...either way, i don't think it's something we need to worry about too much.

LGTM :sass_all_the_things:

File: gulpfile.js (hunk 1 above)
Comment (malucky) [inline gulpfile.js]: everything is already brought in through the `import` so only the files in the root folder is needed

Comment (alexmckenley) [inline gulpfile.js]: I can do that

**Comment by alexmckenley** [comment]
@malucky @deini Should be good to go.

**Comment by malucky** [comment]
:+1:

**Pull request merged**